API/SSW.Rewards.Infrastructure/DependencyInjection.cs

```diff
@@ -30,9 +29,23 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
 
             services.AddScoped<IProfileStorageProvider, ProfileStorageProvider>();
             services.AddScoped<IProfilePicStorageProvider, ProfilePicStorageProvider>();
+            services.AddScoped<IRewardPicStorageProvider, RewardPicStorageProvider>();
 
             services.AddSingleton<IDateTimeProvider, DefaultDateTimeProvider>();
 
+            services.AddScoped<IEmailService, EmailService>();
+
+            services.AddScoped<IRewardSender, RewardSender>();
+
+
+            SMTPSettings smtpSettings = new SMTPSettings();

```

Comment:
Make sure these settings are in the appsettings.json even if they're not the real values

Reply:
done